Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/InteractableToggleCollection.cs

```diff
@@ -29,7 +29,12 @@ private void OnEnable()
                 ToggleList[i].OnClick.AddListener(() => OnSelection(itemIndex));
                 ToggleList[i].CanDeselect = false;
             }
-            
+
```

Comment:
this part can be moved to Awake.
when gameobject is checked in disabled, OnEnable can be invoked before Start.

Reply:
No it has to be in Start. Interactable does work that happens in Awake and need to occur before this code